src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css" />
+		<script src="populated_places.js"></script>
+		<script src="https://unpkg.com/leaflet@1.0.3/dist/leaflet.js"></script>
+
+		<style>
+			body {  
+				padding: 0;
+				margin: 0;
+			}
+
+			html, body, #map{
+				height: 100%;
+				width: 100%;
+			}
+		</style>
+	</head>
+	<body>
+    	<div id='map'></div>
+    	<script>
+      
+      		var map = L.map('map').setView([51.505, -0.09], 4);
+
+      		L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+        		attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+      		}).addTo(map);
+			
+			L.geoJSON(populated_places, {}).addTo(map);	
```

Comment:
yea, that's for sure too much for a browser, just render the capitals or something like that: between 100 and 500 points as maximum